model/project.go

```diff
@@ -137,6 +137,8 @@ type BuildVariantTaskUnit struct {
 	CronBatchTime string `yaml:"cron,omitempty" bson:"cron,omitempty"`
 	// If Activate is set to false, then we don't initially activate the task.
 	Activate *bool `yaml:"activate,omitempty" bson:"activate,omitempty"`
+	// Group is set if an inline task group is defined on the build variant.
+	Group *TaskGroup `yaml:"group,omitempty" bson:"group,omitempty"`
```

Comment:
Can we call this task_group just to be consistent with what users define at the top-level?